AdminServer/appscale/admin/__init__.py

```diff
@@ -221,6 +222,83 @@ class LifecycleState(object):
   DELETE_IN_PROGRESS = 'DELETE_IN_PROGRESS'
 
 
+class AppsHandler(BaseHandler):
+  """ Manages applications. """
+  def initialize(self, ua_client, zk_client):
+    """ Defines required resources to handle requests.
+
+    Args:
+      ua_client: A UAClient.
+      zk_client: A KazooClient.
+    """
+    self.ua_client = ua_client
+    self.zk_client = zk_client
+
+
+  @gen.coroutine
+  def patch(self, name):
```

Comment:
I originally had it as `project_id` but changed it to `name` because of [this](https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps/patch) but I don't have a particular preference

Reply:
"name" in that context is referring the the full resource name (eg. "apps/guestbook").